classes/context/Context.php

```diff
@@ -393,6 +393,63 @@ public function getSupportedLocaleNames(int $langLocaleStatus = LocaleMetadata::
         return $this->getData('supportedLocaleNames') ?? Locale::getFormattedDisplayNames($this->getSupportedLocales(), null, $langLocaleStatus);
     }
 
+    /**
+     * Get the supported added submission locales.
+     *
+     * @return array
+     */
+    public function getSupportedAddedSubmissionLocales()
```

Comment:
@jyhein, I was thinking: it would maybe be good to set those locales already at the install i.e. journal creation, so that these functions can never return null. At the journal creation these locales i.e. the submission default locale can be set to journal primary locale.
There would then always need to be at least one locale selected (as default).
What do you think?

Reply:
Hey @bozana,
I tested this case and seems to do that (when in the install phase I selected English as the primary language, then in the settings wizard English appears in the journal settings > languages > submission languages table; all boxes checked). Is this the place where it is done https://github.com/pkp/pkp-lib/commit/34cd3feac749536f55359a2f22e8b1d2b1d88dfe#diff-4d1a734a7332dda880a48f52a0e281a70a14d011cae5c1f90dc46ca0dc964946 ?

Reply:
Oh, yes @jyhein, thanks!
(I had an empty submission locale list when testing because I have not installed it all anew, and there was no migration script to migrate existing data...)
So yes, then those function should not return null (?array), just array...